tests/core/mempool/test_mempool_manager.py

```diff
@@ -251,5 +260,45 @@ async def get_coin_record(_: bytes32) -> Optional[CoinRecord]:
 
     mempool_manager = await instantiate_mempool_manager(get_coin_record)
     conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1]]
-    _, _, result = await generate_and_add_spendbundle(mempool_manager, conditions)
+    _, _, result = await generate_and_add_spendbundle(mempool_manager, TEST_COIN, conditions)
     assert result == (None, MempoolInclusionStatus.FAILED, Err.UNKNOWN_UNSPENT)
+
+
+@pytest.mark.asyncio
+async def test_sb_twice_with_eligible_coin_and_different_spends_order() -> None:
+    async def get_coin_record(coin_id: bytes32) -> Optional[CoinRecord]:
```

Comment:
if you could come up with a good unique name, this function could also be at global scope, since it only uses global data.

Reply:
Done.